app/src/main/java/com/hacktzi/blog/MainActivity.kt

```diff
@@ -5,8 +5,7 @@ import android.os.Bundle
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
-        //Thread.sleep(10000)//show splashScreen 10 seconds
-        setTheme(R.style.Theme_HackTziBlog) //set normal theme before show activity
+        setTheme(R.style.Theme_HackTziBlog)
```

Comment:
Since Theme_Hacktziblog is the theme for the application, apply it to the application and remove this style from main activity